README.md

```diff
@@ -14,6 +14,30 @@ To start the application, run:
 npm run start
 ```
 
+## Testing
+
+There are two types of tests one for React component as per individual basis and integration for testing different flows. The idea is that component test will be triggered after update to branch excluding **main** and integration for release build or after it has been merged with **main**.
+
+Framework - [CYPRESS](https://docs.cypress.io/)
+
+> Run component tests -> `npm run test:component`
+> Run integration tests -> `npm run test:integration`
+> Plugin configured in `cyopress/plugins/` folder
```

Comment:
This currently renders on a single line and doesn't look so great. Might want to look at reformatting